inference-engine/cmake/features_ie.cmake

```diff
@@ -50,7 +50,7 @@ if (ENABLE_GNA)
     if (UNIX AND NOT APPLE AND CMAKE_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 5.4)
         set (DEFAULT_GNA_LIB GNA1)
```

Comment:
Does this mean that we will have to continue support of GNA1, too? Do we have configurations on CI which involve this branch?

Reply:
We need to support GNA1 for CentOS 7

Reply:
OK, we need to discuss separately if this support is needed - it won't work on new HW anyway.